### PR TITLE
Feature/examples

### PR DESCRIPTION
**Pull request by jdw2465VEN**
Includes my final copy edit of both F5 and Citrix, as well as removal of old hidden comments

### CODE REVIEW

File: examples/citrix_adc/README.md [hunk 1]
```diff
@@ -102,7 +86,9 @@ access_token = "<access_token>"
 venafi_api_key = "<venafi_api_key>"
 ```
 
-And finally configure your Citrix infrastructure (these values are illustrative, you should change them accordingly to your own configutation):
+And finally configure your Citrix infrastructure: 
+
+> **NOTE** The values we use here are for illustration only; you should change them according to your own configuration.
```
Comment (luispresuelVenafi) [inline examples/citrix_adc/README.md:91]: LGTM

File: examples/citrix_adc/README.md [hunk 2]
```diff
@@ -121,7 +107,7 @@ citrix_service_group_members = [ "192.168.6.201:8001", "192.168.6.201:8002", "19
 
 ### Step 2: Set up your main Terraform config file
 
-> **_Important:_** Make sure your local provider is [installed properly](./base/README.md).
+> **IMPORTANT!** If not already done, make sure your local provider is [installed properly](./base/README.md).
```
Comment (luispresuelVenafi) [inline examples/citrix_adc/README.md:110]: LGTM

File: examples/f5_bigip/README.md [hunk 3]
```diff
@@ -310,7 +298,7 @@ f5_pool_members = [ "192.168.6.201:8001", "192.168.6.201:8002", "192.168.6.201:8
     }
     ```
 
-6. Create you resource in order to create your virtual server to manage your F5 partition ([check more aboout this configuration in here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server)):
+6. Create your resource in order to create your virtual server for managing your F5 partition ([check more aboout this configuration in here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server)):
```
Comment (luispresuelVenafi) [inline examples/f5_bigip/README.md:301]: Sorry hehe, `aboout`  it's my typo from my PR

File: examples/f5_bigip/README.md [hunk 4]
```diff
@@ -278,7 +266,7 @@ f5_pool_members = [ "192.168.6.201:8001", "192.168.6.201:8002", "192.168.6.201:8
     }
     ```
 
-4. Create a resource to manages client SSL profiles on a BIG-IP from the F5 partition ([check more aboout this configuration in here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_profile_client_ssl)):
+4. Create a resource to manage client SSL profiles on a BIG-IP from the F5 partition ([learn more aboout this configuration here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_profile_client_ssl)):
```
Comment (luispresuelVenafi) [inline examples/f5_bigip/README.md:269]: Same as https://github.com/Venafi/terraform-provider-venafi/pull/49/files#r610351556

File: examples/f5_bigip/README.md [hunk 5]
```diff
@@ -293,7 +281,7 @@ f5_pool_members = [ "192.168.6.201:8001", "192.168.6.201:8002", "192.168.6.201:8
     }
     ```
 
-5. Create your pool members resources to manage membership in pools ([check more aboout this configuration in here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_pool_attachment)):
+5. Create your pool members' resources to manage membership in pools ([learn more aboout this configuration here](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_pool_attachment)):
```
Comment (luispresuelVenafi) [inline examples/f5_bigip/README.md:284]: Same as https://github.com/Venafi/terraform-provider-venafi/pull/49/files#r610351556

File: examples/f5_bigip/README.md (hunk 3 above)
Comment (luispresuelVenafi) [inline examples/f5_bigip/README.md:301]: Same as https://github.com/Venafi/terraform-provider-venafi/pull/49/files#r610351556

**Comment by luispresuelVenafi** [comment]
In general, the changes looks good to me, I just pointed out some of my typos I could found so they are not missing.

File: examples/f5_bigip/README.md (hunk 3 above)
Comment (jdw2465VEN) [reply examples/f5_bigip/README.md:301]: I'm a writer and I create typos every day (<: 

For the link texts, I suggest you change them to something like "learn more about this configuration here"

File: examples/citrix_adc/README.md (hunk 1 above)
Comment (jdw2465VEN) [reply examples/citrix_adc/README.md:91]: So I'm more efficient at using BitBucket than I am using GitHub. If I do something I shouldn't, please tell me (<:

So once you review edits here, are you going to accept the PR? I'm thinking you and/or Ricardo should do that. And then also do a PR into master for @tr1ck3r to approve?

**Pull request merged**